options.go

```diff
@@ -193,7 +195,6 @@ type Options struct {
 	Compression    int
 	Zoom           int
 	Crop           bool
-	SmartCrop      bool
```

Comment:
This would introduce an API breaking change. Since I don't see value increasing the major version by just this change, I think we should support both options.